modules/video/src/tracking/tracker_vit.cpp

```diff
@@ -45,6 +45,11 @@ class TrackerVitImpl : public TrackerVit
     {
         net = dnn::readNet(params.net);
         CV_Assert(!net.empty());
+
+        if (params.backend != -1 && params.target != -1) {
```

Comment:
Values `-1` are used in case of `#ifndef HAVE_OPENCV_DNN`. So this `if` statement is not actual.

Reply:
Good catch, thank you!